SocialFilter/sections/specific.txt

```diff
@@ -41,6 +41,7 @@ ken-g.com##.entry-footer_share
 haberolduk.com##.kanews-article-share
 mel.fm##.l-publication__mobile-share-button
 destructoid.com###post_sharethis
+sex1014004.pixnet.net##.bookmark
```

Comment:
We have similar rules with `##.bookmark`. Make one rule and I guess `pixnet.net` is better here.